src/cmd/tools/m3db_tool/main/database/cmd.go

```diff
@@ -0,0 +1,65 @@
+package database
+
+import (
+	"flag"
+	"fmt"
+	"go.uber.org/zap"
+	"os"
+
+	"github.com/m3db/m3/src/cmd/tools/m3db_tool/main/common"
+	"github.com/m3db/m3/src/cmd/tools/m3db_tool/main/flagvar"
+	"github.com/m3db/m3/src/query/generated/proto/admin"
+)
+
+var (
+	CmdFlags   *flag.FlagSet
+	createYaml = flagvar.File{}
+
+	defaultPath = "/api/v1/database"
+	debugQS     = "debug=true"
+)
+
+func init() {
```

Comment:
Thanks for the PR!

A couple of high level things:

1. I would recommend avoiding globals--they are always avoidable, and tend to cause more problems than they're worth. Prefer instantiating dependencies in `main`.

2. Could you please write some unittests for this tool? Let me know if you want suggestions on how to do that with the flags--it's usually quite doable.

Reply:
factored the globals, will put in some tests tomorrow

Reply:
@andrewmains12 ok basic tests are in for the cli arg processing and one for the yaml loader since that seems like a new thing